### PR TITLE
Create CCC-TAC-Minutes-2023-03-23.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    * Veraison - Howard Huang - May 18th
+
+***ACTION*** Kurt to revise the Project review slide to show current status
+
+
+**TAC Tech Talk**
+
+* SGX Assurance Tool - Alex Berenzon
+  * Looking for pilot partners, email alex.berenzon@intel.com
+  * Currently does not yet support OE or Linux environments
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:130]:
```suggestion
  * Currently does not yet support OE or Windows environments
```

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+* Quorum was reached and the minutes were approved by the voting members of the TAC.
+
+
+**Action Item Review**
+
+* DM reviewed the action items, proposed spending less time on action
+* KT reported on the white paper conversion to md progress
+* Howard Huang asked for review on project progression state draft PR#160
+* Alec Fernandez asked for review from the TAC ??? on the blog post
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:98]: What is the "???" for?

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  * Currently does not yet support OE or Linux environments
+
+
+**Project Review**
+
+* Open Enclave-SDK - Radhika Jandhyala (RJ)
+  * Growing, up to 164 contributors, 885 stars, 334 forks
+  * DM asked about project progression, Dave Thaler commented that there is not yet an approved next stage
+  * DM discussed if there was anything that the CCC could do to help the project
+  * OP-TEE is not currently being actively maintained, looking for help
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:139]:
```suggestion
  * OP-TEE support is not currently being actively maintained, looking for help
```

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+  
+    ***ACTION*** Thomas to reach out to Linaro to see if they could help with OP-TEE
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:141]:
```suggestion
    ***ACTION*** Thomas to reach out to Linaro to see if they could help with OP-TEE support
```

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+* DM asked if RJ would consider a PR to the Governance Repo to use this review as a template for annual reviews
+  * [Current annual review guidelines](https://github.com/confidential-computing/governance/blob/main/project-progression-policy.md#iv-annual-review-process)
+
+
+**Outreach Update**
+
+* DM reviewed the discussion at GB meeting - DEI policy
+* Graphic on the CCC website - make this diagram correct, currently confusing CC does not include HE for instance and is harmful
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:150]:
```suggestion
* Graphic on the CCC website - make this diagram correct, currently confusing (CC does not include HE for instance) and is harmful
```

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md (hunk 4 above)
Comment (thomas-fossati) [reply TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:141]: "to Linaro _and Scalys_"

Comment (krtaylor) [reply TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md]: Done, added "and Scalys"

**Review by dthaler — APPROVED** [review]

File: TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+* Vidhya Krishnan (NVIDIA)
+* Ahmad Atamli (NVIDIA)
+* Tiziano Santoro (Google)
+* Radhika Jandhyala (Microsoft)
+* Mark Bower (Anjuna)
+* Yash Mankad (Red Hat)
+* Marc Meunier (Arm)
+* Mark Novak (JP Morgan Chase)
+* Sonemaly Phrasavath (AMD)
+* Shalom Shefa
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-03-23/CCC-TAC-Minutes-2023-03-23.md:82]:
```suggestion
* Shalom Shefa (Cisco)
```
Per previous minutes, assuming affiliation is still the same

**Pull request merged**